### DISCUSSION TITLE
playbook filter out experimental rules

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#8149] [Unsupported Versions] [answered]
Hello all,

Is there a way to filter out all sigma rules with a status experimental in playbook?

Regards
Bart

**Comment by defensivedepth** [comment, marked as answer]
Can you give some further context? Filter out from viewing? Filter out from updating? Filter out from generating alerts?

**Reply by sleepingbel** [reply]
in the playbook webview so you can select them
![Knipsel](https://user-images.githubusercontent.com/29328171/174854591-983648c9-9128-4e4a-b86b-9dfc63db8f49.PNG)

But I have seen that they use two values for it
- test
- experimental

![Knipsel](https://user-images.githubusercontent.com/29328171/174855593-b441d52d-90fd-4d11-b66a-a91ccc99ba6d.PNG)
![Knipsel1](https://user-images.githubusercontent.com/29328171/174855645-8346635d-982a-4c63-98dd-37800b8f8296.PNG)

also I have seen the status stable

![Knipsel2](https://user-images.githubusercontent.com/29328171/174855680-e5390f32-4f7e-4195-a367-935581ec3e7a.PNG)

**Reply by defensivedepth** [reply]
ah I see.

So you can do something like the following:

![image](https://user-images.githubusercontent.com/954732/175115236-b4191023-fc3b-4674-9161-29d132d758da.png)

Feel free to create a feature request issue in our repo if you would like to see something else.

**Reply by sleepingbel** [reply]
thx
